timur/test/javascript/utils/query_builder.spec.ts

```diff
@@ -3,40 +3,17 @@ import {QueryGraph} from '../../../lib/client/jsx/utils/query_graph';
 
 const models = {
   monster: {
-    documents: {},
-    revisions: {},
-    views: {},
-    template: require('../fixtures/template_monster.json')
-  },
-  labor: {
-    documents: {},
-    revisions: {},
-    views: {},
-    template: require('../fixtures/template_labor.json')
-  },
-  project: {
-    documents: {},
-    revisions: {},
-    views: {},
-    template: require('../fixtures/template_project.json')
-  },
-  prize: {
-    documents: {},
-    revisions: {},
-    views: {},
-    template: require('../fixtures/template_prize.json')
-  },
-  victim: {
-    documents: {},
-    revisions: {},
-    views: {},
-    template: require('../fixtures/template_victim.json')
-  },
-  wound: {
-    documents: {},
-    revisions: {},
-    views: {},
-    template: require('../fixtures/template_wound.json')
+    documents: {}, revisions: {}, views: {}, template: require('../fixtures/template_monster.json')
```

Comment:
sorry, autoformatter getting aggressive :P  maybe we could write a helper that generates these from the array of required files, sense they are all pretty much empty.  eh, anyways, works for now.